### ISSUE TITLE
questions about multi-tasks 

### ISSUE THREAD

**Issue by linhaow** [zhawe01/fairseq-gec#23]
I didn't find multi-tasks learning in the code , I wonder whether  the code will be released later ?

**Comment by zhawe01** [comment]
No plan now. I can provide the details if you are interested.

**Comment by yingyibiao** [comment]
> No plan now. I can provide the details if you are interested.

I'm very interested!!! Could you kindly provide me with the details of the combined loss design and training details? By the way, where did you store the parameters of the linear transformation from encoder output to sequence label prediction, in the transformer model (in encoder), or the loss function?

**Comment by zhawe01** [comment]
The losses are added directly.
The parameter is defined in the TransformerEncoder:
` if self.use_encoder_classification: 
    self.encoder_cls_linear = nn.Linear(args.encoder_embed_dim, 2)`

**Comment by yingyibiao** [comment]
> The losses are added directly.
> The parameter is defined in the TransformerEncoder:
> ` if self.use_encoder_classification: self.encoder_cls_linear = nn.Linear(args.encoder_embed_dim, 2)`

thanks a lot!
